assets/css/calypsoify.css

```diff
@@ -316,6 +316,105 @@ table.widefat {
 	color: #2E4453;
 }
 
+/* Forms */
+
+#post-body-content {
+	background-color: #fff;
+	box-shadow: 0 0 0 1px rgba(200,215,225,0.5), 0 1px 2px #e9eff3;
+	box-sizing: border-box;
+	padding: 24px;
+}
+
+#title-prompt-text {
+	clip: initial !important; /*avoid clipping when field is on focus */
```

Comment:
Nice work on the clipping property!  If you change the selector here to `#titlediv #title-prompt-text` we can probably remove all of those important tags unless you saw something else overriding these properties.

Reply:
Will do.